sys-fs/reiser4progs/files/reiser4progs-2.0.5-clang.patch

```diff
@@ -0,0 +1,58 @@
+--- a/plugin/vol/volume.c
```

Comment:
Where did this patch come from? It needs some metadata here at a minimum (eg a link to the bug it fixes), see https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#patch-descriptions.

Reply:
How do I specify that `... something you wrote`?

There is noted that it is **possible** to include this description, it is not required

Reply:
There is no way to pay attention of maintainers of Gentoo to any changes to get them merged in, because Gentoo maintainers neither do anything themselves nor do they give the permissions in to merge changes in to someone else
I use all the possibilities I have. If I need to call everyone out on Github to highlight my issue or a problem I will, believe

Reply:
The patch should be sent upstream with a proper commit message before being backported to Gentoo.

Reply:
There is no an upstream active and responsible